### ISSUE TITLE
fbjs removed as React dependency in 16.5.0

### ISSUE THREAD

**Issue by copleykj** [vazco/uniforms#475]
It seems as though `fbjs` has been removed as a dependency from React as of 16.5.0 and upgrading brings the following error.
```
W20181009-21:45:26.834(-4)? (STDERR) Error: Cannot find module 'fbjs/lib/invariant'
W20181009-21:45:26.834(-4)? (STDERR)     at Function.Module._resolveFilename (module.js:547:15)
W20181009-21:45:26.834(-4)? (STDERR)     at Function.Module._load (module.js:474:25)
W20181009-21:45:26.835(-4)? (STDERR)     at Module.require (module.js:596:17)
W20181009-21:45:26.835(-4)? (STDERR)     at require (internal/module.js:11:18)
W20181009-21:45:26.835(-4)? (STDERR)     at Object.<anonymous> (/home/copleykj/projects/meteor/meteor-socialize/node_modules/uniforms/BaseField.js:44:41)
W20181009-21:45:26.835(-4)? (STDERR)     at Module._compile (module.js:652:30)
W20181009-21:45:26.835(-4)? (STDERR)     at Object.Module._extensions..js (module.js:663:10)
W20181009-21:45:26.835(-4)? (STDERR)     at Module.load (module.js:565:32)
W20181009-21:45:26.835(-4)? (STDERR)     at tryModuleLoad (module.js:505:12)
W20181009-21:45:26.836(-4)? (STDERR)     at Function.Module._load (module.js:497:3)
W20181009-21:45:26.836(-4)? (STDERR)     at Module.require (module.js:596:17)
W20181009-21:45:26.836(-4)? (STDERR)     at require (internal/module.js:11:18)
W20181009-21:45:26.836(-4)? (STDERR)     at Object.<anonymous> (/home/copleykj/projects/meteor/meteor-socialize/node_modules/uniforms-bootstrap3/AutoField.js:30:42)
W20181009-21:45:26.836(-4)? (STDERR)     at Module._compile (module.js:652:30)
W20181009-21:45:26.836(-4)? (STDERR)     at Object.Module._extensions..js (module.js:663:10)
W20181009-21:45:26.836(-4)? (STDERR)     at Module.load (module.js:565:32)
W20181009-21:45:26.837(-4)? (STDERR)     at tryModuleLoad (module.js:505:12)
W20181009-21:45:26.837(-4)? (STDERR)     at Function.Module._load (module.js:497:3)
W20181009-21:45:26.837(-4)? (STDERR)     at Module.require (module.js:596:17)
W20181009-21:45:26.837(-4)? (STDERR)     at require (internal/module.js:11:18)
W20181009-21:45:26.837(-4)? (STDERR)     at npmRequire (/home/copleykj/projects/meteor/meteor-socialize/.meteor/local/build/programs/server/npm-require.js:133:10)
W20181009-21:45:26.837(-4)? (STDERR)     at Module.useNode (packages/modules-runtime.js:650:18)
```

**Comment by radekmie** [comment]
Thanks for the report @copleykj. I'm really happy that they finally did. It's on me.

**Comment by radekmie** [comment]
Released in ` v1.27.0-rc.2`.
